socks-forwarder/src/socks/mux.rs

```diff
@@ -80,6 +83,28 @@ impl L2CAPStreamMux {
         mux
     }
 
+    /// Logs (at debug level) the current sending and receiving MTU values.
+    fn log_mtus(&self, socket: &l2cap::Socket<l2cap::Stream>) {
+        match socket.send_mtu() {
+            Ok(smtu) => {
+                debug!("Sending MTU on the connection will be {smtu}");
+            }
+            Err(e) => {
+                debug!("Could not calculate sending MTU; likely not yet negotiated. Error was {e}");
```

Comment:
The error path here and below are not hit from my testing with rock4C+ and rpi4, but I added them in just in case. `send_mtu`, in particular, is documented as liable to error when the value has not yet been negotiated after connection establishment.